pytest/tests/sanity/db_migration.py

```diff
@@ -5,24 +5,27 @@
 Makes sure that the node can still produce blocks.
 """
 
-import json
 import logging
-import os
 import sys
-import time
-import subprocess
-import base58
 import pathlib
 
 sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
 
 import branches
 import cluster
-from transaction import sign_deploy_contract_tx, sign_function_call_tx
+from transaction import sign_deploy_contract_tx, sign_function_call_tx, sign_staking_tx
 import utils
 
 logging.basicConfig(level=logging.INFO)
 
+NUM_SHARDS = 4
+EPOCH_LENGTH = 5
+
+# Config to track all shards.
+node_config = {
+    "tracked_shards": list(range(NUM_SHARDS)),
```

Comment:
lol, this never gets old :)

Reply:
but it makes sense there? if node stops being a validator, we specifically ask it to continue tracking all shards.

Reply:
my point is that any non-empty value for tracked_shards has exactly the same effect

Reply:
I've seen a few different conventions such as setting it to [0] or [666] or [0..num_shard] like you've done, they all mean the same

Reply:
Ah, okay. I didn't know that all these conventions still work